nncf/experimental/onnx/datasets/common.py

```diff
@@ -0,0 +1,68 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import List, Optional, Tuple
+
+from google.protobuf.json_format import MessageToDict
+from nncf.common.utils.logger import logger as nncf_logger
+from onnx import ModelProto
+
+
+def infer_input_shape(model: ModelProto,
```

Comment:
Why is this under the `datasets` folder? I don't think that this function relates to datasets at all. The function should also have a docstring, as per [style guide](https://github.com/openvinotoolkit/nncf/blob/develop/docs/styleguide/PyGuide.md).

Reply:
Took this file out of `datasets` folder and created a docstring.